app/templates/src/app/main/__angular-material.html

```diff
@@ -1,8 +1,9 @@
 <div layout="vertical" layout-fill>
-
-  <header ng-include="'components/navbar/navbar.html'"></header>
-
   <md-content>
+    <md-toolbar>
+      <header ng-include="'components/navbar/navbar.html'"></header>
```

Comment:
This `md-toolbar` also disturb me. There is also one inside the `header.html` template.

I even tried to simply removing this one and it's working fine...